akka-persistence/src/main/scala/akka/persistence/snapshot/local/LocalSnapshotStore.scala

```diff
@@ -26,6 +26,7 @@ import scala.util._
  */
 private[persistence] class LocalSnapshotStore extends SnapshotStore with ActorLogging {
   private val FilenamePattern = """^snapshot-(.+)-(\d+)-(\d+)""".r
```

Comment:
This would mean that I also have to adjust the `SnapshotSeqNrFilenameFilter` which is used in `deleteAsync`. Can do it, of course